modules/va_facilities/spec/requests/nearby_request_spec.rb

```diff
@@ -146,55 +146,88 @@ def query_params(url)
+      end
     end
 
     it 'paginates according to parameters' do
-      get base_query_path + address_params + '&page=2&per_page=3', params: nil, headers: accept_geojson
-      expect(response).to be_success
-      json = JSON.parse(response.body)
-      expect(json['type']).to eq('FeatureCollection')
-      expect(json['features'].length).to eq(3)
-      expect(response.headers['Link']).to be_present
-      parsed = parse_link_header(response.headers['Link'])
-      expect(parsed['self']['page']).to eq(['2'])
-      expect(parsed['first']['page']).to eq(['1'])
-      expect(parsed['last']['page']).to eq(['4'])
-      expect(parsed['prev']['page']).to eq(['1'])
-      expect(parsed['next']['page']).to eq(['3'])
+      setup_pdx
+      VCR.use_cassette('bing/isochrone/pdx_drive_time_60',
+                       match_requests_on: [:method, VCR.request_matchers.uri_without_param(:key)]) do
+        get base_query_path + address_params + '&page=2&per_page=3', params: nil, headers: accept_geojson
+        expect(response).to be_success
+        json = JSON.parse(response.body)
+        expect(json['type']).to eq('FeatureCollection')
+        expect(json['features'].length).to eq(3)
+        expect(response.headers['Link']).to be_present
+        parsed = parse_link_header(response.headers['Link'])
+        expect(parsed['self']['page']).to eq(['2'])
+        expect(parsed['first']['page']).to eq(['1'])
+        expect(parsed['last']['page']).to eq(['4'])
+        expect(parsed['prev']['page']).to eq(['1'])
+        expect(parsed['next']['page']).to eq(['3'])
+      end
     end
 
     it 'paginates empty result set' do
-      get base_query_path + empty_address, params: nil, headers: accept_geojson
-      expect(response).to be_success
-      json = JSON.parse(response.body)
-      expect(json['type']).to eq('FeatureCollection')
-      expect(json['features'].length).to eq(0)
-      expect(response.headers['Link']).to be_present
-      parsed = parse_link_header(response.headers['Link'])
-      expect(parsed['self']['page']).to eq(['1'])
-      expect(parsed['first']['page']).to eq(['1'])
-      expect(parsed['last']['page']).to eq(['1'])
+      setup_pdx
+      VCR.use_cassette('bing/isochrone/pdx_drive_time_1',
+                       match_requests_on: [:method, VCR.request_matchers.uri_without_param(:key)]) do
+        get base_query_path + empty_address, params: nil, headers: accept_geojson
+        expect(response).to be_success
+        json = JSON.parse(response.body)
+        expect(json['type']).to eq('FeatureCollection')
+        expect(json['features'].length).to eq(0)
+        expect(response.headers['Link']).to be_present
+        parsed = parse_link_header(response.headers['Link'])
+        expect(parsed['self']['page']).to eq(['1'])
+        expect(parsed['first']['page']).to eq(['1'])
+        expect(parsed['last']['page']).to eq(['1'])
+      end
+    end
+
+    it 'responds with wait times as part of health services in v1' do
```

Comment:
How painful would it be to add a test case for a facility with no health services? Is that a thing?

Reply:
It turns out there are 46 health facilities that list no health services—a few list no services at all. I wonder if something like that should go in a more general test that doesn't require the cassette like this `/nearby` endpoint.

Reply:
@edmkitty this was painful, but I did finally get a spec added.